ui/StatusQ/src/StatusQ/Components/StatusListItem.qml

```diff
@@ -251,11 +253,25 @@ Rectangle {
                 anchors.leftMargin: 4
             }
 
+            StatusFlatRoundButton {
+                id: errorIcon
+                anchors.top: statusListItemTitle.bottom
+                width: 14
+                height: visible ? 14 : 0
+                icon.width: 14
+                icon.height: 14
+                icon.name: "tiny/warning"
+                icon.color: Theme.palette.dangerColor1
+                tooltip.text: "Error message"
```

Comment:
`qsTr()` missing

Reply:
shouldnt be there int he first place. will remove!